Core/Inc/bmb.h

```diff
@@ -114,13 +121,18 @@ bool initASCI(uint32_t *numBmbs);
 void initBmbs(uint32_t numBmbs);
 
 // TODO update description
-void setGpio(uint32_t numBmbs, uint16_t gpioSetting);
+void cyclicUpdateBMBData(Bmb_S* bmb, uint32_t numBmbs);
 
 // TODO update description
-void readBoardTemps(Bmb_S* bmb,uint32_t numBmbs);
+void updateBMBVoltageData(Bmb_S* bmb, uint32_t numBmbs);
```

Comment:
Generally you should stick to Camel case, even with acronyms. So this should be updateBmbVoltageData